linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/bash/shared.sh

```diff
@@ -18,6 +18,12 @@
 {{% set auditfiles = auditfiles + ["/usr/sbin/audispd"] %}}
 {{% endif %}}
 
+{{% if 'rhel' not in product %}}
+{{% set configString = 'p+i+n+u+g+s+b+acl+selinux+xattrs+sha512' %}}
+{{% else %}}
+{{% set configString = "p+i+n+u+g+s+b+acl+xattrs+sha512" %}}
+{{% endif %}}
+
 {{% set configString = "p+i+n+u+g+s+b+acl+xattrs+sha512" %}}
```

Comment:
I think you'll need to remove definition of `configString` in line 27.

Reply:
Done!